test/helpers/wrappers.go

```diff
@@ -80,7 +80,7 @@ func CurlFail(endpoint string, optionalValues ...interface{}) string {
 		endpoint = fmt.Sprintf(endpoint, optionalValues...)
 	}
 	return fmt.Sprintf(
-		`curl --path-as-is -s -D /dev/stderr --fail --connect-timeout %d --max-time %d %s -w "%s"`,
+		`curl -k --path-as-is -s -D /dev/stderr --fail --connect-timeout %d --max-time %d %s -w "%s"`,
```

Comment:
What's the need to change this here? I don't see any justification in the commit msg.

Reply:
This hunk is changing behavior of a shared function that's used by many tests, so my comment was to raise awareness on that in case there this change introduces unintended behavior changes in other parts of the test suite.